lib/imapoptions

```diff
@@ -1457,6 +1457,9 @@ Blank lines and lines beginning with ``#'' are ignored.
 { "ldap_group_base", "", STRING, "2.3.17" }
 /* LDAP base dn for ldap_group_filter. */
 
+{ "ldap_group_hasmember_attribute", NULL, STRING, "UNRELEASED" }
+/* Specify a field within groups with a list of members. */
+
```

Comment:
Is this not already present as "ldap_member_attribute"?

>            ldap_member_method: attribute
>               Specify  a  group method.  The "attribute" method retrieves groups from a
>               multi-valued attribute specified in ldap_member_attribute.
> 
>               The "filter" method uses a filter, specified  by  ldap_member_filter,  to
>               find  groups;  ldap_member_attribute  is  a  single-value attribute group
>               name.  Allowed values: attribute, filter

Reply:
yep it is :)  I'll fix this up.

Reply:
actually, 'ldap_member_attribute' is 'memberof'; which is different:

```
dn: uid=cassandane,dc=internal
objectclass: user
uid: cassandane
dc: internal
memberof: cn=group co,ou=groups,o=cyrus
memberof: cn=group c,ou=groups,o=cyrus
```

as opposed to:

```
dn: cn=group co,ou=groups,o=cyrus
objectclass: group
cn: group co
ou: groups
o: cyrus
member: uid=cassandane,dc=internal
member: uid=otheruser,dc=internal
```